java/ql/test/library-tests/frameworks/JaxWs/JakartaRs1.java

```diff
@@ -41,7 +41,7 @@ void Post() { // $ ResourceMethod ResourceMethodOnResourceClass
   @Produces("text/plain") // $ ProducesAnnotation=text/plain
   @DELETE
   double Delete() { // $ ResourceMethod=text/plain ResourceMethodOnResourceClass
-    return 0.0; // $ XssSink
+    return 0.0; // $ MISSING: XssSink
```

Comment:
This seems to be an invalid sink due to the `text/plain` content type (see https://github.com/github/codeql/blob/f237360d814afe7eb8a9da963344537d7d3bd31f/java/ql/lib/semmle/code/java/security/XSS.qll#L119-L120).

So we should actually remove the expectations.

```suggestion
    return 0.0;
```